councilmatic_core/models.py

```diff
@@ -233,8 +234,11 @@ def link_html(self):
 
 
 class Post(opencivicdata.core.models.Post):
-    class Meta:
-        proxy = True
+
+    post = models.OneToOneField(opencivicdata.core.models.Post,
+                                on_delete=models.CASCADE,
+                                related_name='councilmatic_post',
+                                parent_link=True)
```

Comment:
`Post` can no longer be a proxy model if we're storing extra `shape` data on it.